### PR TITLE
Added Frigid Optics

### PR DESCRIPTION
**Pull request by FrigidOptics**
I am anNFT artist on Cardano just submitting my project and current policies to be verified for cnft.io marketplace.
https://twitter.com/FrigidOptics/status/1423298836785684482
I don't have a website and all my current policies are already included in the request thing.

### CODE REVIEW

**Comment by itsmestale** [comment]
please read the read me regarding twitter verification

**Comment by FrigidOptics** [comment]
https://twitter.com/FrigidOptics/status/1423298836785684482

File: Frigid Optics [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "52eaa49163b5e8ff229ca75d673751d98a5fbf0ef286be6d0508b915",
+        "ef2ebf4f1dbfa7939f0067a1114d8bb308388f654204fe7e111924a7",
+        "62643a9c7ece3cffffe5a91b4e4cf19deae6037bb3c973fcfe0e6695",
+        "7ee0c1da20c45f95dba6c8055bb2e4b52e6f9afcf98326b599203f17",
+        "eff8bf5db0b4fd9f8208da8e8258f621e7e6e35070ba3798a81a4946",
+        "4a359fe49149e83b1881d38df9b7801f398bf068ff4362881cb1f0de",
+        "b5039c626848b01663206b3e52945c0d55cb528dd4b76c969b951ac6",
+        "e1595be70c27e2029f693efed13d7805abcadb81bc49fe1522cba2cf",
+        "861fc5c802840a84bd3b2a208323e2aff90f6aac3a1d670f036d3a39",
+        "d0215e436e16f03b6fc9e954a06bf79252b281cfd4302613280cc35a",
```
Comment (itsmestale) [inline Frigid Optics:36]: remove last comma from here

**Review by itsmestale — CHANGES_REQUESTED** [review]

File: Frigid Optics (hunk 1 above)
Comment (FrigidOptics) [reply Frigid Optics]: ok done

**Pull request merged**